tests/test_message_sqs.py

```diff
@@ -0,0 +1,31 @@
+"""
+ Test script to:
+  - Validate message sent to Skype channel against the message received on SQS
+"""
+import time
+from conf import sqs_conf
+from conf import skype_conf
+
+def test_message_received_sqs(sqs_instance, skype_instance, concurrent_obj):
```

Comment:
I am not sure I would use sqs instance and skype instances as fixtures. Instead, i would probably use them within some orchestrator in helper. No change needed here though. we can refactor this later.